windows/main_windows.cpp

```diff
@@ -0,0 +1,176 @@
+#include <iostream>
```

Comment:
This whole file deserves a good revamp to use the HTTP API. For the first iteration it's IMHO not necessary though.